specification/datashare/resource-manager/Microsoft.DataShare/stable/2019-11-01/DataShare.json

```diff
@@ -5405,6 +5415,80 @@
         }
       }
     },
+    "StorageAccountDataSet": {
+      "description": "An Azure storage account data set.",
+      "required": [
+        "properties",
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataSet"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/StorageAccountDataSetProperties",
+          "description": "Storage account data set properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "x-ms-discriminator-value": "StorageAccount"
```

Comment:
>StorageAccount [](start = 35, length = 14)

Existing clients will not be able to deserialize the resource contents specific to StorageAccount. Why are you not adding this to a new API version? (This is also being flagged by the breaking change linter check.)